src/yb/tools/yb-admin_client.cc

```diff
@@ -2773,6 +2777,12 @@ Result<rapidjson::Document> ClusterAdminClient::RestoreSnapshotSchedule(
   AddStringField("snapshot_id", snapshot_id.ToString(), &document, &document.GetAllocator());
   AddStringField("restoration_id", restoration_id.ToString(), &document, &document.GetAllocator());
 
+  master_backup_proxy_->ListSnapshotSchedules(list_req, &list_resp, &rpc);
+  if(!list_resp.has_error()) {
+    auto table_name = list_resp.schedules(0).options().filter().tables().tables()[0].table_name();
```

Comment:
The path the code in `SnapshotScheduleInfoToJson` walks down the nested proto messages is:
```
schedule.options().filter().tables().tables(0).namespace_().name()
```

In your PR we do:
```
schedule.options().filter().tables().tables(0).table_name()
```

I'm not sure why you get `table_name()` instead of `namespace_().name()`. Does the`table_name` field contain the name of the database? Even if it does, it's clearer to use `namespace_().name()` in both places.

Reply:
Great catch! My bad, I think I just got confused. Thank you for your patience I will fix that!